src/pages/index.module.css

```
@@ -1,23 +1,394 @@
-/**
- * CSS files with the .module.css suffix will be treated as CSS modules
- * and scoped locally.
- */
+/* ----- Elements ------  */
 
-.heroBanner {
-	padding: 4rem 0;
+:root {
+	--pink: #f37;
+	--blue: #2563eb;
+	--bg-blue: #e0f2fe;
+}
+
+.main code {
+	font-size: 0.85rem !important;
```

```suggestion
	font-size: 0.8rem !important;
```

Saving horizontal space